tests/pages/components/course-page/course-stage-step/second-stage-instructions-card.ts

```diff
@@ -2,7 +2,12 @@ import { clickable, collection, isVisible, text } from 'ember-cli-page-object';
 
 export default {
   clickOnCompleteStepButton: clickable('[data-test-complete-step-button]'),
+  clickOnExpandStepButton: clickable('[data-test-expand-step-button]'),
+  clickOnRevealSolutionButton: clickable('[data-test-reveal-solution-button]'),
+
```

Comment:
@andy1li let's not add newlines between statements like this - for reference, the rule we follow is: we only add newlines around statements that are multi-line (like `scrollIntoView` below). Everything else is just sorted alphabetically. More consistenty => Less cognitive load required + easier to automate refactoring / use AI for changes. 

The existing empty line between `scope` and `hasScreencastsLink` looks like a mistake - feel free to correct that.